test/integration/test_open_analyse_save_load_project.c

```diff
@@ -37,7 +37,7 @@ bool test_open_analyse_save() {
 	size_t functions_count_expect = rz_list_length(functions);
 
 	// 4. Save into the project
-	if (!rz_file_exists(".tmp/")) {
+	if (!rz_file_is_directory(".tmp/")) {
```

Comment:
is this folder even legal for windows? @ret2libc

Reply:
It is - test was passing on Windows

Reply:
Yeah I think it is nowadays.. anyway, fixed on another PR.